lib/backend/etcdv3/inline_cert_key.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2019 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Don't think we need two copies of the Apache license.  Suggest you order the file like this:
* Tigera copyright
* Explanation of etcd copyright
* etcd copyrights
* Apache license section

Reply:
Acknowledged and updated license structure.